entity/scenario_intersection/include/scenario_intersection/utility.h

```diff
@@ -14,7 +15,7 @@ template <typename T>
 T convert(const std::string&);
 
 template <typename T, typename F>
-decltype(auto) if_exist(const YAML::Node& node, const std::string& key, F&& consequent)
+decltype(auto) if_exist(const YAML::Node& node, const std::string& key, F&& consequent, rclcpp::Logger & logger)
```

Comment:
Why not have this: 
```suggestion
decltype(auto) if_exist(const YAML::Node& node, const std::string& key, F&& consequent, const rclcpp::Logger & logger)
```

Same with `Controller`,

Reply:
Done